src/content/reference/app-defaulting-validation/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Defaulting and Validation of app CRs
+description: How defaulting and validation of app CRs is implemented by app-admission-controller
+weight: 110
+type: page
+last_review_date: 2020-12-16
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+---
+
+## Overview
+
+For Giant Swarm releases using app-operator version 3.0.0 and upwards the
+defaulting and validation logic of [app CRs]({{< relref "/content/reference/cp-k8s-api/apps.application.giantswarm.io.md" >}}) is enabled. This logic is provided by
+[app-admission-controller](https://github.com/giantswarm/app-admission-controller).
+
+We have not enabled the defaulting and validation for existing app CRs to avoid
+disrupting your existing usage of App Platform.
```

Comment:
The missing "the" in "usage of App Platform" sounds a bit strange to me. But I'm not the native speaker. Just pointing it out for alignment.

Reply:
I think it reads better as the "the App Platform" I'll fix.